extensions/ql-vscode/src/data-extensions-editor/queries/java.ts

```diff
@@ -148,6 +146,82 @@ boolean isSupported(CallableMethod method) {
   or
   not method.isSupported() and result = false
 }
+
+// The below is a copy of https://github.com/github/codeql/blob/249f9f863db1e94e3c46ca85b49fb0ec32f8ca92/java/ql/lib/semmle/code/java/dataflow/internal/ModelExclusions.qll
+// to avoid the use of internal modules.
+/** Holds if the given package \`p\` is a test package. */
+pragma[nomagic]
+private predicate isTestPackage(Package p) {
```

Comment:
It is odd that checking whether a file is a test file is public in C# and private in Java. It seems we should be consistent on that matter?

Reply:
I had hoped they would both be public, but while we can use the Java `ModelExclusions` file, it didn't seem right to use an internal module. I agree that it's odd that there's a difference between the two languages.